recommendation_engine/config/path_constants.py

```diff
@@ -1,20 +1,16 @@
 """Contains the path constants for S3 and local storage."""
 import os
 
-ECOSYSTEM = os.environ.get('CHESTER_SCORING_REGION', 'npm')
 MODEL_VERSION = os.environ.get('MODEL_VERSION', '2019-01-03')
-DEPLOYMENT_PREFIX = os.environ.get('DEPLOYMENT_PREFIX', 'dev')
```

Comment:
Did you mean to remove all these variables here?

Reply:
Yes because of new AWS directory changes.